bee-network/examples/actor.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#[tokio::main]
+async fn main() {
+    std::env::set_var("RUST_LOG", "debug");
+    env_logger::init();
+
+    startup().await.unwrap();
+}
+
+async fn startup() -> anyhow::Result<()> {
+    std::panic::set_hook(Box::new(|info| {
+        log::error!("{}", info);
+    }));
+
+    // RuntimeScope::<ActorRegistry>::launch(|scope| {
```

Comment:
What's the status of this example ? If it does nothing let's just remove it.

Reply:
Removed.